Bottom: 34850c3fae2002c32308b3353855e2de0a18b4d3
Top:    9e26e65c6f5ad1ae860b4b03c429a90054cbb837
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-13 22:12:28 -0300

Thread safety documentation.

for ChangeLog

	* manual/startup.texi: Document thread safety properties.


---

diff --git a/manual/startup.texi b/manual/startup.texi
index a277714..7bbaf13 100644
--- a/manual/startup.texi
+++ b/manual/startup.texi
@@ -358,8 +358,8 @@ value is nonzero and @code{errno} is set to indicate the error.
 The difference to the @code{setenv} function is that the exact string
 given as the parameter @var{string} is put into the environment.  If the
 user should change the string after the @code{putenv} call this will
-reflect in automatically in the environment.  This also requires that
-@var{string} is no automatic variable which scope is left before the
+reflect automatically in the environment.  This also requires that
+@var{string} not be an automatic variable whose scope is left before the
 variable is removed from the environment.  The same applies of course to
 dynamically allocated variables which are freed later.
